mono/mini/mini-runtime.c

```diff
@@ -1990,7 +1990,7 @@ mono_jit_compile_method_with_opt (MonoMethod *method, guint32 opt, gboolean jit_
 	MonoJitICallInfo *callinfo = NULL;
 	WrapperInfo *winfo = NULL;
 
-	error_init (error);
+	//error_init_internal (error); // FIXME
```

Comment:
Why not just removing it?

Reply:
Still trying to determine which are needed.